infrastructure/simulation-tool/test/simulations/deposit.test.ts

```diff
@@ -0,0 +1,33 @@
+import { Wallet as RollupWallet, RootstockOperation } from '@rsksmart/rif-rollup-js-sdk';
+import { expect, use } from 'chai';
+import { BigNumber } from 'ethers';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import { depositToSelf } from '../../src/operations/deposit';
+
+use(sinonChai);
+
+describe('depositToSelf', () => {
```

Comment:
why is this a simulation and not an operation?

Reply:
moved